packages/compass-home/src/components/home.tsx

```diff
@@ -124,16 +125,14 @@ function reducer(state: State, action: Action): State {
 }
 
 function hideCollectionSubMenu() {
-  void import('hadron-ipc').then(({ ipcRenderer }) => {
-    if (ipcRenderer) {
-      ipcRenderer.call('window:hide-collection-submenu');
-    }
-  });
```

Comment:
@gribnoysup is there a reason we were loading this asynchronously? Wondering if I can change this behavior, or re-add it.

Reply:
IIRC @mcasimir made all hadron-ipc imports dynamic awhile ago in all packages so that the tests don't break when you are running them outside of electron runtime. This is just following this pattern